### ISSUE TITLE
Brackets around footnote anchors

### ISSUE THREAD

**Issue by charliehoward4dp** [DistributedProofreaders/ppwb#25]
The old standalone ppcomp had a way of recognizing that square brackets around footnote anchors in plain text should not be reported (when there aren't corresponding brackets around superscripted anchors in HTML). Can an option for that be added to this version?

**Comment by rtonsing** [comment]
Can you point me to some example files?

**Comment by charliehoward4dp** [comment]
"Secrets of Polar Travel" [https://www.pgdp.net/c/project.php?id=projectID5eb5e47871e45](url) is in Smooth Reading at DP. The Plain Text and HTML have only two footnotes, so it'll be easy to see what happens.

**Comment by rtonsing** [comment]
@cpeel Can this be moved to ppcomp? I don't have access to assign it (not that I necessarily should).

**Comment by cpeel** [comment]
It can't be moved to a different repo (ie `ppcomp`) without closing it and opening a new one. If you want to open a copy at `ppcomp` I can close this one, or I'm happy to assign this ticket to you and we can work it under this issue -- either are fine with me.

**Comment by rtonsing** [comment]
Actually, this can already be handled by the example in the Readme, by passing `--css '.fnanchor:before { content: "["; } .fnanchor:after { content: "]"; }'`.
 The problem is that the PP Workbench site does not provide a way of entering custom --css arguments. @cpeel: Could this be added?
The [pptools](https://pptools.tangledhelix.com/) site provides an example.

**Comment by charliehoward4dp** [comment]
Until recently, I used ppcomp at the pptools website, where custom css is supported. Unfortunately, the next version of Guiguts (1.4.x) generates HTML5, which the pptools website cannot handle.  The PP Workbench does handle HTML5, but its ppcomp tool doesn't support custom css.  (It also does not allow custom css for thought breaks, but mostly seems to handle them anyway: only one was flagged, and only once (the text and HTML files have many thought breaks, and all match, including the flagged one).

I probably should file a separate bug report for the following (please advise):  When the"Extract and process footnotes separately" option is selected, the Workbench's ppcomp returns this error message:

`Whoops! Something went wrong and no output was generated. The error message was


For more assistance, ask in the [discussion topic](https://www.pgdp.net/phpBB3/viewtopic.php?f=13&t=64838) and include this identifier: r625483e56d486`

(There is no error message.)  If you need a pair of test files for any of these issues, they're attached below.  This may be related to file being in HTML5.

[ppcomp_html5_limitation.zip](https://github.com/DistributedProofreaders/ppwb/files/8467259/ppcomp_html5_limitation.zip)

**Comment by cpeel** [comment]
> The problem is that the PP Workbench site does not provide a way of entering custom --css arguments. @cpeel: Could this be added?

It can, yes. Please open an Issue here in the `ppwb` repo.

**Comment by rtonsing** [comment]
> Until recently, I used ppcomp at the pptools website, where custom css is supported. Unfortunately, the next version of Guiguts (1.4.x) generates HTML5, which the pptools website cannot handle. 

I have offered the updated code to the owner of Tangledhelix, but never heard back. I get the impression it is not really maintained any more. I was able to try your example by changing the 1st line to the HTML4 style.

> The PP Workbench does handle HTML5, but its ppcomp tool doesn't support custom css. (It also does not allow custom css for thought breaks, but mostly seems to handle them anyway: only one was flagged, and only once (the text and HTML files have many thought breaks, and all match, including the flagged one).

Requested separately.
 
> I probably should file a separate bug report for the following (please advise): When the"Extract and process footnotes separately" option is selected, the Workbench's ppcomp returns this error message:
> 
> `Whoops! Something went wrong and no output was generated.

Yes, please enter a new issue in the ppcomp repo. I know the footnotes extraction is poor and am working on it, but there are so many different styles it will never be foolproof. Shouldn't do a "Whoops!" at least.

**Comment by rtonsing** [comment]
> It also does not allow custom css for thought breaks, but mostly seems to handle them anyway: only one was flagged, and only once (the text and HTML files have many thought breaks, and all match, including the flagged one).

The one flagged actually has **`<tb>`** in the text file instead of asterisks. Good catch?

**Comment by charliehoward4dp** [comment]
Blush

Thank you; yes, good catch.

**Comment by srjfoo** [comment]
> I have offered the updated code to the owner of Tangledhelix, but never heard back.

The pptools code was written by user Bibimbop;`tangledhelix` is owned by user dlowe, who offered to host it for DP's use. See [here](https://www.pgdp.net/phpBB3/viewtopic.php?p=1231655#p1231655) and [here](https://www.pgdp.net/phpBB3/viewtopic.php?p=936928#p936928).
